cmd/create/accountroles/cmd.go

```diff
@@ -133,6 +140,22 @@ func init() {
 		"Create only classic Rosa account roles",
 	)
 
+	flags.StringVar(
```

Comment:
@gdbranco / @hunterkepley What's the expected flag names? Looking at https://issues.redhat.com/browse/OCM-12291, i see `–cross-account-hosted-zone-role-arn` and `–cross-account-vpc-endpoint-service` which is different to what we have here.

Reply:
Thanks. 

@hunterkepley I'd suggest updating the description of https://issues.redhat.com/browse/OCM-12291 just to make things clearer when we get to QE etc.